tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py26, py27, py33, py34
+envlist = py27, py36
```

Comment:
This feels like a much bigger change than the scope of this PR. I'm not against it (2.7 & 3.6 are definitely the right targets) but might there be any undesirable side-effects of this change?

Reply:
Ahh, I leave my tox like that and normally ignore adding it, it's creeped in.

I think it will be fine, it makes sense in the long run as those are the versions that Travis uses and currently Travis doesn't run tox. Worst case it's easy file to revert